ast/builtins.go

```diff
@@ -1078,7 +1078,7 @@ var ParseDurationNanos = &Builtin{
 var Date = &Builtin{
 	Name: "time.date",
 	Decl: types.NewFunction(
-		types.Args(types.N),
+		types.Args(types.A),
```

Comment:
Can you change these to use the `types.NewAny(..)` helper like in https://github.com/open-policy-agent/opa/blob/master/ast/builtins.go#L528-L544 with only the `types.N` and `types.NewArray(..)`

It helps make the definition more clear as to what values are allowed.

Reply:
Ah, got it. That's what I was looking for, sorry I was a bit slow to find it.